Makefile

```diff
@@ -15,9 +16,14 @@ test:
 
 lint:
 	${LINT} run --build-tags=all
+	${FORMATTER} -d core
+	if [[ -n `${FORMATTER} -d core` ]]; then exit 1; fi
```

Comment:
We don't really need to run this twice here, do we?
OTOH, perhaps this was a test to make sure I was paying attention :joy:

Reply:
Yes, you've passed the test! 😂  
Jokes aside, I found that the easiest and most reliable way to print the diff emitted by the formatter (it prints nothing if the code is correct) and exit with code 1 if there is something to change. Unfortunately the formatter always exit with 0.

Reply:
Okay, I've changed it to call it only once, let me know what you think! :)